src/PerformanceTests/Tests/Categories/PublishVsSendFixture.cs

```diff
@@ -0,0 +1,49 @@
+namespace Categories
+{
+    using System.Collections.Generic;
+    using NUnit.Framework;
+    using Tests.Permutations;
+    using Variables;
+
+    [TestFixture(Description = "Publish vs Send", Category = "Performance")]
+    public class PublishVsSendFixture : Base
```

Comment:
Why is this recreated? The `PublishVsSendFixture` was renamed to `TransportsFixture`.